### DISCUSSION TITLE
file is not a zipfile

### DISCUSSION THREAD

**Discussion by tommypank** [kubilus1/autoortho#67] [General]
Hey man, still not able to use this program but I keep trying. I just downloaded the new pre release and I'm getting this error

INFO:downloader:ERROR: File is not a zip file with Zipfile C:\Users\tompa\.autoortho-data\downloads\z_aus_pac_01.zip. 

Any advice? Trying not to download the whole of aus again

**Comment by kubilus1** [comment]
Looks like a corrupt zip.  So you can try the latest pre-release.  Run that
once and you should see a new option in your config file (Users dir
.autoortho file) to set 'noclean = True'.

That will prevent deletion of existing downloads.  You may need to manually
delete the possibly corrupt zip though.

Give that a shot!

**Reply by tommypank** [reply]
Worked, thank you!

**Reply by Acidstuff** [reply]
I'm on 0.4.3 and having the same issue . The files are downloaded but as soon as it starts unpacking it reduces the file-size to 1 kb.

"_INFO:downloader:ERROR: File is not a zip file with Zipfile C:\Users\gert-\.autoortho-data\downloads\z_eur_00.zip.  Recommend retrying_"

![image](https://github.com/kubilus1/autoortho/assets/887144/9411df78-37be-4826-b0bd-128a928eace9)

**Reply by MaximilianUhlig** [reply]
I have the exact same issue as Acidstuff. My folder looks the exact same. After you click 'Ok' on the error window that tells you to retry it seems to delete the z_eur_00.zip file and all files have a file size of 1KB.

**Comment by kubilus1** [comment]
Wait a bit and retry, and it should work.  You may need to manually clear
out your downloads folder first though.
